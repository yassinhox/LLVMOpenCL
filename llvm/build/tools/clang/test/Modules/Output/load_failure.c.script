rm -rf /home/jazouani/llvm/build/tools/clang/test/Modules/Output/load_failure.c.tmp &&
/home/jazouani/llvm/build/Release+Asserts/bin/clang -cc1 -internal-isystem /home/jazouani/llvm/build/Release+Asserts/bin/../lib/clang/3.2/include -fmodules -x objective-c++ -fmodule-cache-path /home/jazouani/llvm/build/tools/clang/test/Modules/Output/load_failure.c.tmp -fdisable-module-hash -emit-module -fmodule-name=load_failure /home/jazouani/llvm/llvm-3.2/tools/clang/test/Modules/Inputs/module.map &&
/home/jazouani/llvm/build/Release+Asserts/bin/clang -cc1 -internal-isystem /home/jazouani/llvm/build/Release+Asserts/bin/../lib/clang/3.2/include -fmodules -x objective-c -fmodule-cache-path /home/jazouani/llvm/build/tools/clang/test/Modules/Output/load_failure.c.tmp -fdisable-module-hash /home/jazouani/llvm/llvm-3.2/tools/clang/test/Modules/load_failure.c -DNONEXISTENT 2>&1 | FileCheck -check-prefix=CHECK-NONEXISTENT /home/jazouani/llvm/llvm-3.2/tools/clang/test/Modules/load_failure.c &&
not /home/jazouani/llvm/build/Release+Asserts/bin/clang -cc1 -internal-isystem /home/jazouani/llvm/build/Release+Asserts/bin/../lib/clang/3.2/include -fmodules -x objective-c -fmodule-cache-path /home/jazouani/llvm/build/tools/clang/test/Modules/Output/load_failure.c.tmp -fdisable-module-hash /home/jazouani/llvm/llvm-3.2/tools/clang/test/Modules/load_failure.c -DFAILURE 2> /home/jazouani/llvm/build/tools/clang/test/Modules/Output/load_failure.c.tmp.out &&
FileCheck -check-prefix=CHECK-FAILURE /home/jazouani/llvm/llvm-3.2/tools/clang/test/Modules/load_failure.c < /home/jazouani/llvm/build/tools/clang/test/Modules/Output/load_failure.c.tmp.out
