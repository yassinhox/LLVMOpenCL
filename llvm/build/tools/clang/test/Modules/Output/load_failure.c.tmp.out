error: C99 was disabled in PCH file but is currently enabled
1 error generated.
